### PR TITLE
Conglomerate pagure events.

### PR DESCRIPTION
**Pull request by ralphbean**
Around tickets, PRs, and users who commit.

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/conglomerators/pagure/pagure.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import fedmsg.meta.base
+from fedmsg_meta_fedora_infrastructure.fasshim import avatar_url, email2fas
+
+
+
+
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/conglomerators/pagure/pagure.py]: Still want to follow pep8? :)

File: fedmsg_meta_fedora_infrastructure/conglomerators/pagure/pagure.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        tmpl['link'] = link_template.format(
+            repo=repo,
+            entity_name=self.entity_name,
+            idx=idx,
+        )
+
+        return tmpl
+
+
+class ByPR(AbstractPagureTicketConglomerator):
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/conglomerators/pagure/pagure.py]: I like this approach with the parent class inherited by each conglomerators :)

**Comment by pypingou** [comment]
:+1: for me but do I read this correctly that we don't have tests for it? Is that intended?

**Comment by ralphbean** [comment]
> do I read this correctly that we don't have tests for it?

Nope.  They are here:  https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure/blob/feature/pagure-conglomeration/fedmsg_meta_fedora_infrastructure/tests/conglomerate/pagure/test_pagure.py

github says that "the diff is to large to show here".

File: fedmsg_meta_fedora_infrastructure/conglomerators/pagure/pagure.py (hunk 2 above)
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/conglomerators/pagure/pagure.py]: Yah, me too!

**Comment by ralphbean** [comment]
Thanks @pypingou!

**Pull request merged**